storage/src/test/java/tech/pegasys/teku/storage/server/leveldb/LevelDbInstanceTest.java

```diff
@@ -227,6 +230,31 @@ void stream_performUnsignedComparisonsOnKeys() {
     }
   }
 
+  @Test
+  void putRaw_shouldWriteToStorage() {
+    // random data is fine, but must not have signed bit set because of the 'valueOf' logic
+    final Bytes byteData = Bytes.random(8).and(Bytes.fromHexString("0x7FFFFFFFFFFFFFFF"));
+    final UInt64 data = UInt64.valueOf(byteData.toLong());
+    update(tx -> tx.putRaw(variable3, byteData));
+    assertThat(instance.get(variable3)).contains(data);
+    assertThat(instance.getRaw(variable3)).contains(byteData);
+  }
+
+  @Test
+  void streamRaw_shouldGetByteData() {
+    // random data is fine, but must not have signed bit set because of the 'valueOf' logic
+    final Bytes byteData = Bytes.random(8).and(Bytes.fromHexString("0x7FFFFFFFFFFFFFFF"));
+    final UInt64 data = UInt64.valueOf(byteData.toLong());
+    update(tx -> tx.put(column4, UInt64.ZERO, data));
+
+    assertThat(instance.get(column4, UInt64.ZERO)).contains(data);
+    try (final Stream<ColumnEntry<Bytes, Bytes>> stream = instance.streamRaw(column4)) {
+      assertThat(stream.map(ColumnEntry::toString))
+          .containsExactly(
+              ColumnEntry.create(Bytes.fromHexString("0x0000000000000000"), byteData).toString());
```

Comment:
Not sure we should be comparing the `toString` results here.  Probably need to either implement equals/hashCode on `ColumnEntry` or use assertj's ability to compare field by field.